### PR TITLE
create the influxdb client

### PR DESCRIPTION
**Pull request by BolaNasr**

### CODE REVIEW

File: 0-os/templates/statistics/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## template: Statistics gathering 
+
+### Description:
+This template is responsible for get statistics of node then add it in InfluxDB to Visualization in grafana
+### Schema:
+
+- `instanceName`: instance name that created from influxdb_Client template .
```
[the remark this answers is not in the document]
Comment (BolaNasr) [reply 0-os/templates/statistics/README.md]: Done

File: 0-os/templates/statistics/statistics.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import os
+import datetime
+import json
+from js9 import j
+from zerorobot.service_collection import ServiceNotFoundError
```
[the remark this answers is not in the document]
Comment (BolaNasr) [reply 0-os/templates/statistics/statistics.py]: Done

File: 0-os/templates/influxdb_client/influxdb_client.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import os
+import datetime
+import json
+from js9 import j
```
[the remark this answers is not in the document]
Comment (BolaNasr) [reply 0-os/templates/influxdb_client/influxdb_client.py]: Done

File: 0-os/templates/influxdb_client/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+## template: influxdb_Client
+
+### Description:
+This template is responsible for create influxdb client 
+
+### Schema:
+
+- `instanceName`: Name of the instance
```
Comment (zaibon) [inline 0-os/templates/influxdb_client/README.md]: you don't need to have an instance name in the schema. use the name of the service itself as instance name of the client

Comment (BolaNasr) [reply 0-os/templates/influxdb_client/README.md]: ok , but when i create database at statistics Template should get the name of service it self to get influxdb_client ?

Comment (zaibon) [reply 0-os/templates/influxdb_client/README.md]: yes. So you create the influxdb_client service then you create the statistics one and in the data you give the name of the influxdb_client so it knows what to use

File: 0-os/templates/statistics/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (zaibon) [reply 0-os/templates/statistics/README.md]: I don't see any difference, did you forget to push?

File: 0-os/templates/statistics/statistics.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (zaibon) [reply 0-os/templates/statistics/statistics.py]: I don't see any difference, did you forget to push?

**Pull request merged**